blocked-edges/4.12.41-OVNKubeMasterDSPrestop.yaml

```diff
@@ -1,5 +1,5 @@
 to: 4.12.41
-from: .*
+from: 4[.](11[.].*|12[.]([0-9]|[1-3][0-9]|40)[+].*)
```

Comment:
I don't understand the literal `+` part (`[+]`) at the end - should that be a `-` instead to cover the RCs and ECs in the `4.12.0-rc.6` form?

Reply:
[Docs][1]:

> The `[+].*` portion absorbs [the architecture-suffix](https://github.com/openshift/cincinnati-graph-data/tree/b927aa2f6c38ad5e9a8aa195211e9f2c8d7df542#release-names) from the release name that consumers will use for comparisons.

So basically it's making sure the `[0-9]` case doesn't match the first digit of `4.12.43` and similar, by requiring the next character to be the `+` of the arch-suffix, and not a 3 or other digit.

Yes, it will exclude the `-` of RCs and ECs, but all of these are far enough past GA that they shouldn't have direct-from-candidate update paths.

[1]: https://github.com/openshift/cincinnati-graph-data/tree/b927aa2f6c38ad5e9a8aa195211e9f2c8d7df542#block-edges

Reply:
OK, TIL, I have never seen these before :monocle_face: